actionpack/test/controller/integration_test.rb

```diff
@@ -1199,6 +1199,34 @@ def post_to_foos(as:)
     end
 end
 
+module ActiveRecord
```

Comment:
Short of moving the entire test into railties, I couldn't figure out a way of making the check work correctly without defining this here.